go/libkb/interfaces.go

```diff
@@ -613,6 +613,13 @@ type TeamLoader interface {
 	ClearMem()
 }
 
+type DeviceEKStorage interface {
+	Put(ctx context.Context, generation keybase1.EkGeneration, deviceEK keybase1.DeviceEk) (err error)
```

Comment:
I'm surprised @oconnor663 let you get away with `DeviceEk` rather than `DeviceEphemeralKeyInOtherWordsAKeyThatIsNotPersisted` :)

Reply:
I consider it a great victory that the whackronym D.E.K. will never appear in our code.